instrumentation/apache-dubbo-2.7/library-autoconfigure/src/main/java/io/opentelemetry/instrumentation/apachedubbo/v2_7/DubboTelemetry.java

```diff
@@ -35,8 +36,13 @@ public static DubboTelemetryBuilder builder(OpenTelemetry openTelemetry) {
     this.clientInstrumenter = clientInstrumenter;
   }
 
-  /** Returns a new Dubbo {@link Filter} that traces Dubbo RPC invocations. */
-  public Filter newFilter() {
```

Comment:
Usually we mark the old method as deprecated, keep it around for at least one release, and then remove.

Reply:
> Usually we mark the old method as deprecated, keep it around for at least one release, and then remove.

I find it often appears in some situations where a change from experimental to stable. This is a bug here, I think we shouldn't allow users to use the method again. 
BTW, if it's necessary here, I found `TracingFilter` might need to keep the original `invoke` implementation again. It's a bit strange.

Reply:
one of the reasons we deprecate first before removing (when possible) is that we use the `@deprecated` Javadoc to tell users which method to use instead. you can remove the method in the following minor release

Reply:
Thanks for your reminder and explanation, I have adjusted it and please have a look after holiday.